Source/Core/VideoCommon/DriverDetails.cpp

```diff
@@ -58,6 +58,7 @@ namespace DriverDetails
 		{OS_WINDOWS,VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_LINUX,  VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_WINDOWS,VENDOR_INTEL,    DRIVER_INTEL,        -1, BUG_INTELBROKENBUFFERSTORAGE, 101810.3907, 101810.3960, true},
+		{OS_ALL,    VENDOR_MESA,     DRIVER_I965,         -1, BUG_BROKENIMAGELOADSTORE,-1.0, -1.0, true},
```

Comment:
Was the bug reported upstream? 
If so it would have been handy to have that entry cross-referenced here so, had this fix been temporarily applied, it then could have been reverted at time of fix by upstream.